code/game/turfs/turf.dm

```diff
@@ -21,10 +21,10 @@
 
 	var/explosion_level = 0	//for preventing explosion dodging
 	var/explosion_id = 0
-	
+
 	var/list/decals
 
-/turf/New()
+/turf/Initialize()
 	..()
 
 	levelupdate()
```

Comment:
On second thought, this should be done in another PR. I wonder if smoothing the zlevel is even wise for it's initialize since everything forces a smooth queuing it should be fine to just remove the zlevel smoothing bit from SSicon_smooth's initialize.

Then you can make SSicon_smooth much much faster by adding a flag to smooth for if something is already queued to smooth, and check that rather then do an associated add.